sky/cli.py

```diff
@@ -2559,17 +2559,12 @@ def autostop(
 
     - The cluster has restarted.
 
-    - An autostop is set when there is no active setting. (Namely, either
-      there's never any autostop setting set, or the previous autostop setting
-      was canceled.) This is useful for restarting the autostop timer.
+    - A new autostop idle time is set.
```

Comment:
"new": What if there was an existing N minutes, and users submit an autostop command with N again?

Reply:
Removed "new" to be more clear. Thanks!